### PR TITLE
mlayer-geoQuery-API

### PR DESCRIPTION
**Pull request by tanvi029**

### CODE REVIEW

File: src/main/java/iudx/catalogue/server/database/Constants.java [hunk 1]
```diff
@@ -136,6 +137,11 @@ public class Constants {
           "{\"_source\":[$2],\"query\":{\"bool\": {\"must\": [ { \"match\": {\"domainId.keyword\":\"$1\"} } ] } } }";
   public static final String GET_MLAYER_PROVIDERS_QUERY =
           "{\"query\": {\"match\": {\"type.keyword\": \"iudx:Provider\"}},\"_source\": {\"includes\": [\"id\",\"description\"]}}";
+  public static final String POST_MLAYER_GEOQUERY =
+      "{ \"query\": { \"bool\": { \"minimum_should_match\": 1, \"should\": [$1]}},\"_source\": {\"includes\": [\"id\",\"location\",\"instance\",\"label\"] }}";
+  public static final String POST_MLAYER_BOOL_GEOQUERY =
```
Comment (pranavrd) [inline src/main/java/iudx/catalogue/server/database/Constants.java:142]: POST_XX_QUERY can be changed to GET_XX_QUERY as they are fetch ops

File: src/main/java/iudx/catalogue/server/database/ElasticClient.java [hunk 2]
```diff
@@ -380,11 +394,27 @@ public void onSuccess(Response response) {
...
+                    JsonObject location = source.getJsonObject("location");
+                    String label = source.getString("label");
+                    String dataset_id = source.getString("id");
+                    String doc_id = results.getJsonObject(i).getString(DOC_ID);
+                    JsonObject result =
+                        new JsonObject()
+                            .put(INSTANCE, instance)
+                            .put("dataset_id", dataset_id)
+                            .put("location", location)
+                            .put("label", label)
```
Comment (pranavrd) [inline src/main/java/iudx/catalogue/server/database/ElasticClient.java:409]: Can use result.mergIn(source) instead

**Comment by pranavrd** [comment]
ok to test

**Review by kailash — APPROVED** [review]
LGTM

**Pull request merged**